doc/nrf/config_and_build/configuring_app/index.rst

```diff
@@ -76,5 +76,343 @@ Just as for creating the application, you can build the application using either
    kconfig/index
    advanced_building
    output_build_files
+   zephyr_samples_sysbuild
 
 .. |output_files_note| replace:: For more information about files generated as output of the build process, see :ref:`app_build_output_files`.
+
+.. _sysbuild_images:
+
+Sysbuild images
+###############
```

Comment:
We cannot have this kind of structure on a container page.
Please move this added content to a separate page.

Reply:
@FrancescoSer asked for it like this, so which should it be?